homeassistant/components/workday/binary_sensor.py

```diff
@@ -208,3 +218,8 @@ async def async_update(self):
 
         if self.is_exclude(day_of_week, date):
             self._state = False
+
+    @property
+    def unique_id(self) -> str:
```

Comment:
Use the `self._attr_unique_id` instance attribute.
https://developers.home-assistant.io/docs/core/entity/#entity-class-or-instance-attributes